packages/web-app-admin-settings/src/components/Groups/GroupsList.vue

```diff
@@ -28,7 +28,11 @@
           :label="$gettext('Select all groups')"
           :model-value="allGroupsSelected"
           hide-label
-          @update:model-value="$emit('toggleSelectAllGroups')"
+          @update:model-value="
+            allGroupsSelected
+              ? $emit('unSelectAllGroups')
```

Comment:
is it possible to extract the event into a const?

Reply:
why?

Reply:
same reason why strings are often put into constants: to make it harder to mess up typing the string correctly. :trollface: referencing a variable is always easier than looking up the correct string...

Reply:
I see... we use them multiple time. We can make this happen in this pr, even if this is not a refactoring pr 😅

So we also wanted to keep it as small as possible for you guys to review.

But you get what you asked for 😜🖤